src/dom/dom.ts

```diff
@@ -849,16 +847,14 @@ export class dom {
      *
      * ```
      */
-    static getStyle(el: Element): CSSStyleDeclaration {
-        return el['currentStyle'] || window.getComputedStyle(el);
+    static getStyle(el: Element): CSSStyleDeclaration {return el['currentStyle'] || window.getComputedStyle(el);
```

Comment:
Revert this change?

Reply:
Apologies, must've hit a wrong key there.